File: 1960-check-if-the-sentence-is-pangram/1960-check-if-the-sentence-is-pangram.java
class Solution {
    public boolean checkIfPangram(String sentence) {
        int[] ascii = new int[256];
        for(char c : sentence.toCharArray()){
            ascii[c]++;
        }
        for(int i = 97; i <= 122; i++){
            if(ascii[i] == 0) return false;
        }
        return true;
    }
}

File: 1960-check-if-the-sentence-is-pangram/1960-check-if-the-sentence-is-pangram.cpp
class Solution {
public:
    bool checkIfPangram(string sentence) {
        vector<int> ascii(256, 0);
        for (char c : sentence) {
            ascii[c]++;
        }
        for (int i = 97; i <= 122; i++) {
            if (ascii[i] == 0) return false;
        }
        return true;
    }
};
